apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -293,7 +293,7 @@ var (
 			WeaveNode:                 m("weaveworks/weave-kube:2.1.2"),
 			WeaveCNI:                  m("weaveworks/weave-npc:2.1.2"),
 			PodInfraContainer:         m("gcr.io/google_containers/pause-amd64:3.1"),
-			Ingress:                   m("rancher/nginx-ingress-controller:0.10.2-rancher2"),
+			Ingress:                   m("rancher/nginx-ingress-controller:0.16.2-rancher1"),
```

Comment:
@orangedeng we should update rancher ingress image only for the system images/k8s versions that were not released prior to 2.0.7. The images are retrieved in dynamic manner from rancher, and if they are different from whatever is deployed, the cluster will get recreated